types/jsonschema-protobuf/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Type definitions for jsonschema-protobuf 1.0
+// Project: https://github.com/karissa/jsonschema-protobuf
+// Definitions by: Mo Kweon <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/jsonschema-protobuf/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me

Reply:
Thank you for catching.
Fixed.